benefits/eligibility/templates/eligibility/includes/selection-label--calfresh.html

```diff
@@ -0,0 +1,18 @@
+{% extends "eligibility/includes/selection-label.html" %}
+{% load i18n %}
+
+{% block label %}
+    {% translate "CalFresh Cardholder" %}
+{% endblock label %}
+
+{% block description %}
+    {% translate "You must have" %}
+    {% comment %} modal-trigger needs to be refactored to support translating the text {% endcomment %}
```

Comment:
Interesting :thinking: 

Would you want to do that in this PR, since this is the first time it is being introduced? We don't have an issue for this otherwise.

Reply:
I realized we don't actually need any refactor 😅 We can pass in the translated value, which is what I've done in [1a52320](https://github.com/cal-itp/benefits/pull/1958/commits/1a523202565c234688e1696dee87b37fdab34da6)

Django also probably wouldn't have been able to detect a variablized `{% translate text %}` in `modal-trigger`, so it's better to do it directly in the specific template rather than in the includes.


_From https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#standard-translation_
> (The caveat with using variables or computed values, as in the previous two examples, is that Django’s translation-string-detecting utility, [django-admin makemessages](https://docs.djangoproject.com/en/5.0/ref/django-admin/#django-admin-makemessages), won’t be able to find these strings. More on [makemessages](https://docs.djangoproject.com/en/5.0/ref/django-admin/#django-admin-makemessages) later.)